src/locales/ja.yaml

```
@@ -281,36 +335,46 @@ app:
         status: 状態
         total_duration: 合計期間
     title:
+      add_chart: グラフを追加
       add_printer: プリンターの追加
       bedmesh: ベッドメッシュ
       bedmesh_controls: ベッドメッシュコントロール
       camera: カメラ｜カメラ
       config_files: Configurationファイル
       configure: Configuration
       console: コンソール
+      diagnostics: 診断
+      edit_chart: チャートを編集
       endstops: エンドストップ
       fans_outputs: ファン＆出力
-      gcode_preview: 'Gcode Preview'
+      gcode_preview: 'Gcode Preview [ ベータ版 ]'
```

This isn't beta anymore.
```suggestion
      gcode_preview: 'Gcode Preview'
```